docs/developers/v2/fork-yearn-ui.md

```diff
@@ -0,0 +1,219 @@
+# Fork Yearn UI
+
+This is a step-by-step guide on how to fork and customize [macarena.finance](https://macarena.finance/) which is a yearn UI made to be forked using the open-source [repository](https://github.com/yearn/macarena-finance). Deploying your own UI makes you eligible to receive partner [profit-sharing](https://docs.yearn.finance/partners/introduction#profit-share-model) fees.
+
+  ## Install required software
+
+To follow all steps you need to install the following dependencies:
+
+1. **git**
+    - https://git-scm.com/downloads
+2. **node.js**
+    - https://nodejs.org/en/download/
+3. Open a terminal window and set up Yarn (used to install dependencies)
+    - open terminal **Windows**: Windows + R -> type `cmd` -> Enter
+    - open terminal **Mac**: CMD + Space -> type `Terminal` -> Enter
+    - set up yearn: `npm install --global yarn`
+
+## Clone Macarena Finance
+
+1. Fork Macarena Finance
+    - https://github.com/yearn/macarena-finance/fork
+2. Open terminal and clone your fork
+    - `git clone git@github.com:YOUR_GITHUB_NAME/macarena-finance.git`
+    - `cd macarena-finance` to enter project folder after cloning
+3. Install dependencies 
+    - `yarn install`
+4. Run developer environment
+    - `yarn dev`
```

Comment:
this is not the same as README.md. There is says `yarn run dev`

Reply:
both would work but I will standardize to one, thanks!